### DISCUSSION TITLE
Informal chat

### DISCUSSION THREAD

**Discussion by tbillington** [Rust-GPU/rust-gpu#5] [Q&A] [answered]
Just wondering if there are any places to discuss that are "official" as of the new organisation, since previous discussion was in the embark discord.

**Comment by nihalpasham** [comment]
+1 

is there a plan to add a matrix or discord channel to the README page?

**Comment by LegNeato** [comment, marked as answer]
We're thinking we want to use GitHub discussions for a bit and see how that goes. Discord had a lot of good info that was not discoverable...we're hoping that by having it in GitHub discussions more people can find existing answers.

**Reply by nyabinary** [reply]
I would suggest Matrix instead of Discord if Github Discussion aren't good enough :P
